src/Symfony/Component/Validator/Constraints/ExpressionValidator.php

```diff
@@ -44,6 +44,7 @@ public function validate($value, Constraint $constraint)
 
         if (!$this->getExpressionLanguage()->evaluate($constraint->expression, $variables)) {
             $this->context->buildViolation($constraint->message)
+                ->atPath((string) $constraint->errorPath)
```

Comment:
the cast to string is not needed

Reply:
looks like it's needed https://github.com/symfony/symfony/pull/41921/checks?check_run_id=2971187070#step:8:510